lib/engine/step/buy_sell_par_shares.rb

```diff
@@ -179,10 +183,15 @@ def pass!
         end
       end
 
-      def can_buy_multiple?(_entity, corporation, _owner)
+      def can_buy_multiple?(_entity, corporation, owner)
+        if @game.multiple_buy_only_from_market?
+          return false unless owner.share_pool?
+          return false if @round.bought_from_ipo
+        end
+
         corporation.buy_multiple? &&
-         @round.current_actions.none? { |x| x.is_a?(Action::Par) } &&
-         @round.current_actions.none? { |x| x.is_a?(Action::BuyShares) && x.bundle.corporation != corporation }
+          @round.current_actions.none? { |x| x.is_a?(Action::Par) } &&
```

Comment:
stray change? undo.

Reply:
It was indented with one space instead of two. Is that right?

Reply:
yep, it's unchanged